pom.xml

```diff
@@ -854,23 +578,10 @@
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>buildnumber-maven-plugin</artifactId>
-                <version>1.2</version>
```

Comment:
version is 1.3 in parent pom. Not necessarily a problem but might be worth checking?